src/App.js

```diff
@@ -85,9 +87,11 @@ class App extends Component {
   };
 
   render() {
+    const { inTeams, taskId, shareTag, user, loading, error } = this.state;
     return (
       <div className="App" style={{ backgroundImage: `url(${background})` }}>
-        {!this.state.loading && !this.state.error ? (
+        <Debug />
```

Comment:
Is this `<Debug />` element still needed? What does it do?

Reply:
Yes. Debug gives you the ability to easily see the app context with the click of a small debug icon: 
<img width="348" alt="image" src="https://user-images.githubusercontent.com/76968885/222856265-6d4094cc-a636-4117-830f-ba20a73867a2.png">

Reply:
That's cool!